doc/basic_pv_pvc_dv.md

```diff
@@ -0,0 +1,200 @@
+## Kubernetes Storage Overview
```

Comment:
Thanks for reworking that, this section is better IMO.  Personally I'd leave most of the detail out of it, but I know you want to include it and I think others do as well so that makes sense.

It gets a little tricky because of dynamic-hostpath, hostpath, local, dynamic-local-provisioner, the CSI plugins etc etc etc

Reply:
The goal is to have something for people to kick the tires with, storage is complicated, and the amount of permutations is huge, so a couple of 'simple' examples to get folks going is all I am aiming for.